packages/modules/electricity_tariffs/voltego/tariff.py

```diff
@@ -3,6 +3,7 @@
 import logging
 from typing import Dict
 import pytz
+import requests
```

Comment:
Von dem kompletten Modul wird lediglich eine Fehlerklasse benötigt. Macht es nicht Sinn, dann auch nur diese zu importieren?
```suggestion
from requests.exceptions import HTTPError
```